.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,4 +1,2 @@
-
-Closes ISSUE #NUMBER (identify the issue associated with this PR)
-
-**Describe your PR**
+We are moving to a new home. (https://github.com/golang/vscode-go).
```

Comment:
Maybe just:

We are moving to a new home. Please send your PR to https://github.com/golang/vscode-go instead.

Reply:
I agree

Reply:
Fixed in https://github.com/microsoft/vscode-go/pull/3237/commits/25800e3ad930142159ac813eac4cabcff6b99ed3